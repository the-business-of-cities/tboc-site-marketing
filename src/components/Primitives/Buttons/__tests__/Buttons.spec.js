import {Button, IconButton} from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Button", () => {
  it("renders correctly", () => {
    expect(shallow(<Button />)).toMatchSnapshot();
  });
});

describe("IconButton", () => {
  it("renders correctly", () => {
    expect(shallow(<IconButton />)).toMatchSnapshot();
  });
});
