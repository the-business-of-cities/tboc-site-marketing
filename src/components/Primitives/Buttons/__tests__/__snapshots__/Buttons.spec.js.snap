// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<MaybeLink>
  <styled.div />
</MaybeLink>
`;

exports[`IconButton renders correctly 1`] = `
<MaybeLink>
  <styled.div>
    <span />
  </styled.div>
</MaybeLink>
`;
